src/StatusQ/Components/StatusToastMessage.qml

```diff
@@ -0,0 +1,166 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+import QtGraphicalEffects 1.13
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+
+Control {
+    id: root
+    width: 343
+    height: !!secondaryText ? 68 : 48
+    anchors.right: parent.right
```

Comment:
Should the anchors be handled by the component that uses it instead?

Reply:
this is to handle the animation for show/hide. Atm I don't have a view of how that should be working since very limited info is available on Figma. This should be re-visited when we know how toast messages should work exactly